### DISCUSSION TITLE
Cüzdan kodları

### DISCUSSION THREAD

**Discussion by Mrt021** [IBAX-io/go-ibax#159] [Ideas]
Var olan cüzdanın puplic keyine nerden ulaşabiliriz acaba

**Comment by hrhrrcp** [comment]
Kardeş bu maillleri nasıl iptal edicem sürekli geliyor

iPhone’umdan gönderildi

Mrt021 ***@***.******@***.***>> şunları yazdı (23 Oca 2022 16:34):


Var olan cüzdanın puplic keyine nerden ulaşabiliriz acaba

—
Reply to this email directly, view it on GitHub<https://github.com/IBAX-io/go-ibax/discussions/159>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXL3UYYYJAUFAUPTPV6XRLDUXP7XDANCNFSM5MTLKG3Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by 11eren** [comment]
cüzdan kısmında düzelmesi gereken şeyler var

**Comment by Gazarta** [comment]
Cüzdaninizi Twitter bağlandıktan sonra sol üstteki 3 çizgiye bastıktan sonra assagidaki profil fotoğrafınızı basarak cüzdanın üstüne tıklayın orda çıkar

**Reply by Mrt021** [reply]
Teşekkür ederim 🙏🙏
